modeling-cmds/src/ok_response.rs

```diff
@@ -160,6 +160,12 @@ define_ok_modeling_cmd_response_enum! {
             /// Curve type
             pub curve_type: CurveType,
         }
+        /// The response from the `SetClearColor` command.
+        #[derive(Debug, Serialize, Deserialize, JsonSchema, Eq, PartialEq, ExecutionPlanValue, ModelingCmdOutput)]
+        pub struct SetClearColor {
+            /// Successfully set the clear color
+            pub success: bool,
```

Comment:
There's no need for a response here, all this response is doing is indicating if there was an error or not, and we already show errors to the user. If the command fails, the user will get an error the usual way through our error schema. Take a look at how the other commands respond with errors.

Reply:
thanks for explaining this. I have removed it now